superset/views/core.py

```diff
@@ -857,10 +857,6 @@ def dashboard_permalink(
     def log(self) -> FlaskResponse:
         return Response(status=200)
 
-    @expose("/theme/")
```

Comment:
@geido Just to confirm, is this not needed for server-side rendered pages?

Reply:
@michael-s-molina I think that was just an initial experiment to build an internal storybook, not really related to the theme.